lib/gui/pages/settings/templates/settings.tpl.html

```diff
@@ -46,7 +46,11 @@ <h3 class="subtitle">Advanced</h3>
     <label>
       <input type="checkbox"
         ng-model="settings.currentData.unsafeMode"
-        ng-change="settings.enableDangerousSetting('unsafeMode', 'Are you sure you want to turn this on? You will be able to burn to your system drives.')">
+        ng-change="settings.enableDangerousSetting('unsafeMode', {
+          message: 'Are you sure you want to turn this on? You will be able to burn to your system drives.',
```

Comment:
Yeah, "overwrite" might sound better; as both "write to" and "destroy data on" make it sound like it'll possibly only write to **part** of the disk, whereas obviously as soon as it's written over the MBR the whole disk is lost (as far as normal users are concerned).
And :+1: to the " if you are not careful" on the end.